fhirstarter/tests/conftest.py

```diff
@@ -1,27 +1,38 @@
 """FHIRStarter test fixtures"""
 
 import pytest
-from requests.models import Response
+from _pytest.fixtures import FixtureRequest
 
 from ..testclient import TestClient
-from .config import client, client_create_and_read
-from .utils import resource
+from .config import create_test_client
+
+
+@pytest.fixture(scope="session", params=[True, False], ids=["async", "nonasync"])
+def async_endpoints(request: FixtureRequest) -> bool:
+    """Parametrized fixture to ensure that all tests are tested in both async and nonasync modes."""
+    return request.param
```

Comment:
Took me a while to figure out how fixture parametrization works, but this and the `client` fixture below are the key to making all tests run in both modes.